.ci/deps.sh

```diff
@@ -67,8 +67,8 @@ fi
 
 deps_perl="perl libperl-critic-perl"
 
-sudo apt-get -y update
-sudo apt-get -y --no-install-recommends install $deps $deps_perl $deps_infer
+sudo apt-get -y update|grep -v '\%'
```

Comment:
@yukiisbored I am afraid you misinterpreted I didn't mean it would harm log files, I was asking if setting term environment to dumb will have any effect on log files at all as it changes the environment variable of terminal window not the log file.